src/lwc/utilExpandableSection/utilExpandableSection.js

```diff
@@ -7,12 +7,13 @@ export default class utilExpandableSection extends LightningElement {
     @api isCollapsed = false;
     @api alternativeText;
     @api bodyClass;
+    @api buttonBackground;
```

Comment:
We removed this component from the UI per a UX requested change - I'll roll these changes to this file back.